src/class/SignToken/SignToken.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-bitwise */
+export default class SignToken {
+  /**
+   * @param {string} str
+   * @return {number}
+   */
+  getHash(str) {
+    let hash = 4678;
+    let i = str.length;
+
+    while (i) {
+      i -= 1;
+      hash = (hash * 33) ^ str.charCodeAt(i);
+    }
+
+    /* JavaScript does bitwise operations (like XOR, above) on 32-bit signed
+     * integers. Since we want the results to be always positive, convert the
+     * signed int to an unsigned by doing an unsigned bitshift. */
+    return hash >>> 0;
+  }
+
+  /**
+   * @param {*} obj
```

Comment:
Remove `any` type everywhere.
Don't see `@ts-check`

Reply:
ts-check was added and here and in the next method above we really can accept any type and return any type (number, string, object, array)